facia-purger/cloudformation/cloudformation.json

```diff
@@ -115,7 +115,65 @@
           ]
         },
         "Runtime": "java8",
-        "Timeout": 20
+        "Timeout": 30
+      }
+    },
+    "NotificationTopic": {
+      "Type": "AWS::SNS::Topic",
+      "Properties": {
+        "Subscription": [ {
+          "Protocol": "email",
+          "Endpoint": "dotcom.platform@guardian.co.uk"
+        } ]
+      }
+    },
+    "InvocationAlarm": {
+      "Type": "AWS::CloudWatch::Alarm",
+      "Properties": {
+        "AlarmActions": [
+          { "Ref": "NotificationTopic" }
+        ],
+        "InsufficientDataActions": [
+          { "Ref": "NotificationTopic" }
+        ],
+        "AlarmDescription": "Notify if there are less than 5 invocations over last 5 minutes or there is insufficient data (i.e. no invocations)",
+        "ComparisonOperator": "LessThanThreshold",
+        "Dimensions": [
+          {
+            "Name": "FunctionName",
+            "Value": { "Ref": "Lambda" }
+          }
+        ],
+        "EvaluationPeriods": "1",
+        "MetricName": "Invocations",
+        "Namespace": "AWS/Lambda",
+        "Period": "300",
+        "Statistic": "Sum",
+        "Threshold": "5",
```

Comment:
Those values seems fine when you assume all the upstream systems work as intended. However if facia-press or the fronts editorial tools go down, this alarm will be triggered even though there is nothing wrong with it.

Since this alarm is not critical (ie: fronts are not broken, they would only be slower to refresh) I would use much higher values (higher than the "SLA" of facia-press or any editorial tools) so we are informed the purger stopped working without adding more noise when upstream systems are not working as intended.

I am not sure I am very clear. Let's discuss face to face @jfsoul